pytealutils/__init__.py

```diff
@@ -0,0 +1,6 @@
+from .inline import *
+from .iter import *
+from .math import *
+from .storage import *
+from .strings import *
+from .transaction import *
```

Comment:
I'd like to suggest again that this file remain empty to avoid name collisions and exporting things that were not meant to be public.

Was there a reason this was added back?

Reply:
i honestly dont remember doing it 😆

Reply:
Probably because I'm doing something wrong but I needed it for relative imports for some reason. I can mess around and figure out what is wrong with the way I'm importing